src/main/java/appeng/integration/BaseModule.java

```diff
@@ -21,12 +21,6 @@
 
 public abstract class BaseModule implements IIntegrationModule
 {
-
-	protected void testClassExistence( Class<?> clz )
-	{
-		clz.isInstance( this );
-	}
-
 	@Override
 	public abstract void init() throws Throwable;
```

Comment:
The craftguide integration needs to extend `CraftGuideAPIObject` and thus extending `BaseModule` is simply not possible (except we would switch to scala).

We could move it into the `IIntegrationModule` interface, but then there is a chance that it will be missed when refactoring it later. Keeping it in a helper class will reduce this possibility.

I am also not positive about the current `testClassExistence` implemenation as is uses an reflective `instanceof`. `Class.forName(clz.getName())` might be a more fitting implementation and remove the need to pass this around. It really just needs to throw an exception.
Maybe `clz.getName()` could already throw one.